CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # astro changelog
 
-## 0.5.0 (UNRELEASED, 2019)
-
+## 0.5.0 (October 3, 2019)
```

Comment:
Now might be a good idea to adopt the format from https://keepachangelog.com, specifically:

```
### Added
* New ls command to tvm (#37)
* etc

### Changed
* Adopt options pattern for `astro.NewProject` constructor (#26)
* etc
```

Where each of the changes fits into a heading named:

## Types of changes
* `Added` for new features.
* `Changed` for changes in existing functionality.
* `Deprecated` for soon-to-be removed features.
* `Removed` for now removed features.
* `Fixed` for any bug fixes.
* `Security` in case of vulnerabilities.

-- from https://keepachangelog.com/en/1.0.0/

Reply:
Updated, pls take another look